app/uk/gov/hmrc/trackingconsentfrontend/assets/.eslintrc.json

```diff
@@ -0,0 +1,47 @@
+{
+  "extends": [
+    "airbnb-base"
```

Comment:
Did you try using eslint-config-airbnb-typescript: https://www.npmjs.com/package/eslint-config-airbnb-typescript? It might simplify this configuration.

Reply:
I'll take a look.

Reply:
I've made an update which now uses that.

Reply:
Wouldn't you need to do ```extends: ['airbnb-typescript'],``` in your .eslintrc.json?